modules/ROOT/pages/_partials/task-add-api-instance.adoc

```diff
@@ -98,7 +98,10 @@ See xref:runtime-fabric::proxy-deploy-runtime-fabric.adoc[Deploying API Proxies
 * Click **Select API from Exchange** if you have an API shared with you through Exchange that you want to manage.
 [arabic]
 .. Click the API from the list under **Select API**. You can search for a specific API if needed.
-.. Update the *Asset type*, *API version*, and *Asset version* if needed.
+.. Update the *Asset type*, *API version*, and *Asset version* if you are not using the latest version.
++
+For more information about versions in Exchange, see xref:exchange::asset-versions.adoc[].
```

Comment:
Add ^ to indicate that the link is going outside of docs.mulesoft.com.

Reply:
The numbering for this task is whacked in the git preview. You might want to double-check it in a local build.